eoscostarica/evodex
# Social media preview

### GH Metadata tags

### What does this PR do?
- Show social media preview
- Create HelmetMetaData component
- Delete PageTitle component
- Refactor languages files

### Steps to test
1. yarn start
2. go to localhost:3000
3. navigate between pages and see tab title, also share Evodex and see social preview

#### CheckList
